libraries/rig-package/src/RigConfig.ts

```diff
@@ -337,6 +337,56 @@ export class RigConfig {
     return this._resolvedProfileFolder;
   }
 
+  /**
+   * This lookup first checks for the specified relative path under `projectFolderPath`; if it does
+   * not exist there, then it checks in the resolved rig profile folder.  If the file is found,
+   * its absolute path is returned. Otherwise, `undefined` is returned.
+   *
+   * @remarks
+   * For example, suppose the rig profile is:
+   *
+   * `/path/to/your-project/node_modules/example-rig/profiles/example-profile`
+   *
+   * And suppose `configFileRelativePath` is `folder/file.json`. Then the following locations will be checked:
+   *
+   * `/path/to/your-project/folder/file.json`
+   *
+   * `/path/to/your-project/node_modules/example-rig/profiles/example-profile/folder/file.json`
+   */
+  public tryResolveConfigFilePath(configFileRelativePath: string): string | undefined {
+    const localPath: string = path.join(this.projectFolderPath, configFileRelativePath);
+    if (fs.existsSync(localPath)) {
+      return localPath;
+    }
+    if (this.rigFound) {
+      const riggedPath: string = path.join(this.getResolvedProfileFolder(), configFileRelativePath);
```

Comment:
```suggestion
      const riggedPath: string = path.resolve(this.getResolvedProfileFolder(), configFileRelativePath);
```

Reply:
`this.getResolvedProfileFolder()` returns an absolute path.

Reply:
🤔 But maybe `configFileRelativePath` is not nicely relative.  Seems like we should add some validation to forbid that.

Reply:
Okay I added logic in https://github.com/microsoft/rushstack/pull/2232/commits/64a67dab51d3bb18afaebf88925b9cb7ffccbde8 which validates that `configFileRelativePath` will not require `path.resolve()`